typpete/inference_runner.py

```diff
@@ -90,7 +94,10 @@ def run_inference(file_path=None):
     if not os.path.exists(write_path):
         os.makedirs(write_path)
 
-    file = open(write_path + '/{}_constraints_log.txt'.format(file_name), 'w')
+    if write_path.endswith('/'):
```

Comment:
Do these things work on Windows as well?

Reply:
Haven't tried, I don't have a windows machine. Why do you think it might be different?

Reply:
Because windows uses backslashes  in paths where Linux/Mac use normal slashes. Maybe we should do it like this: https://stackoverflow.com/questions/16010992/how-to-use-directory-separator-in-both-linux-and-windows-in-python

Reply:
Well I think the whole project then is not compatible with Windows (especially `import_handler` and `stubs_handler`, we use Linux slashes there everywhere). If we want to support windows in this aspect, we need to change it in many other places, and if so, I prefer to do it in another pull request, since it is off the scope of this pull request.